app/backend/controllers/login-signup.js

```diff
@@ -59,3 +69,30 @@ module.exports.login = async (request, response) => {
     }
   }
 }
+
+/*
+  Get endpoint for verify email.
+*/
+module.exports.verify = async (request, response) => {
+  const token = request.query.token
+  try {
+      let userRegistered = await User.findOne({ token })  // Retrieve the user instance from database
+      if (!userRegistered) {  // If no instance is returned, credentials are invalid
+        throw Error('User not found.')
+      }
+
+      userRegistered.isVerified = true
+      userRegistered.token = ""
```

Comment:
We don't have to delete verification token. If the user try to re-verify his account, we can tell him that his account is already verified that way.